electron-app/package.json

```diff
@@ -23,26 +23,26 @@
     }
   },
   "dependencies": {
-    "@theia/core": "next",
-    "@theia/filesystem": "next",
-    "@theia/workspace": "next",
-    "@theia/preferences": "next",
-    "@theia/navigator": "next",
-    "@theia/process": "next",
-    "@theia/terminal": "next",
-    "@theia/editor": "next",
-    "@theia/languages": "next",
-    "@theia/markers": "next",
-    "@theia/monaco": "next",
-    "@theia/messages": "next",
-    "@theia/vsx-registry": "next",
-    "@theia/keymaps": "next",
-    "@theia/getting-started": "next",
+    "@theia/core": "latest",
+    "@theia/filesystem": "latest",
+    "@theia/workspace": "latest",
+    "@theia/preferences": "latest",
+    "@theia/navigator": "latest",
+    "@theia/process": "latest",
+    "@theia/terminal": "latest",
+    "@theia/editor": "latest",
+    "@theia/languages": "latest",
+    "@theia/markers": "latest",
+    "@theia/monaco": "latest",
+    "@theia/messages": "latest",
+    "@theia/vsx-registry": "latest",
+    "@theia/keymaps": "latest",
+    "@theia/getting-started": "latest",
     "@theia/electron": "next",
```

Comment:
I think this one needs to be `latest` as well?

Reply:
I had a quick look and this might work as-is, only because `@theia/electron` is not itself pulling other Theia extensions. But still preferable to have all Theia extensions the same: `latest` or `next`.

Reply:
I think I see why you might have kept this one on `next`. We've updated Electron recently on master, from 4.x to 9.x. Since you've been using `next`, that change has already landed in this repo and in `yarn.lock`. By adopting `latest` right now, you're rolling-back Electron to 4.x, against your current `yarn.lock`. 

I would suggest to keep things as they are now, and wait until Thursday for the Theia release. At that point, `@theia/electron:latest` will also be using `Electron 9.x`, so you can switch and I think everything should fall back on its feet.

Reply:
yes, I  missed that

Reply:
I'll update this PR on Thursday when @theia/electron/latest will use Electron 9.x

Reply:
I'll give it another try after the update